../../media/renderers/video_frame_rgba_to_yuva_converter.h:33:19: error: no template named 'optional' in namespace 'std'
   33 | MEDIA_EXPORT std::optional<gpu::SyncToken> CopyRGBATextureToVideoFrame(
      |              ~~~~~^
1 error generated.

--- a/media/renderers/video_frame_rgba_to_yuva_converter.h
+++ b/media/renderers/video_frame_rgba_to_yuva_converter.h
@@ -9,6 +9,8 @@
 #include "media/base/media_export.h"
 #include "third_party/skia/include/gpu/ganesh/GrTypes.h"
 
+#include <optional>
+
 namespace gfx {
 class Size;
 }  // namespace gfx
